api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java

```diff
@@ -1644,4 +1644,14 @@ public String runAllAutomatedReportsAndPostToDHIS() {
 	public List<LocationToOrgUnitMapping> getAllLocationToOrgUnitMappings() {
 		return getDao().getAllLocationToOrgUnitMappings();
 	}
+
+	@Override
+	public void saveLocationToOrgUnitMapping(LocationToOrgUnitMapping locationToOrgUnitMapping) {
```

Comment:
thanks @Piumal1999  could you create tests for the added methods

Reply:
@HerbertYiga Ok sure. I'll update the pull request.

Reply:
Added the tests with https://github.com/openmrs/openmrs-module-dhisconnector/commit/16e8fd2c29edb38bcfbae751bf338cddf1735eb5